defaults/initrd.scripts

```diff
@@ -1226,16 +1228,29 @@ write_env_file() {
 }
 
 crypt_filter() {
+	[ -z "${LUKS_KEY}" -o "$(echo ${LUKS_KEY} | grep -o '.gpg$')" = ".gpg" ]
```

Comment:
I forgot `LUKS_KEY` is local in `openLUKS()`. Although of course it will work as is. But you have to pass it by argument.
`local luks_key="${1}"`

Reply:
Will be in next commit/push.